lib/rubycritic/source_locator.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'pathname'
+require 'rubycritic/configuration'
```

Comment:
Why this require?

Reply:
If I remember correctly, the tests failed without this when I ran them outside of `rake` like
```
bundle exec ruby -Ilib:test test/lib/rubycritic/generators/json_report_test.rb 
```
Personally, I also consider it good style to have Ruby files be "self-sufficient", i.e. you can `require` them without having to `require` other files first, but that might be a matter of taste.